### PR TITLE
Typos in code

### PR DESCRIPTION
**Pull request by ss18**
## Motivation
To fix typos

## Test Plan
No testing required, changed only internal things names

## Related PRs
No

## Release Notes
addtionalStyles -> additionalStyles
occured -> occurred
recomendedType -> recommendedType
markDirtyAndPropogate -> markDirtyAndPropagate
targetting -> targeting
RCTApplyTranformationAccordingLayoutDirection -> RCTApplyTransformationAccordingLayoutDirection (tranform -> transform)

### CODE REVIEW

File: ReactCommon/yoga/yoga/YGNode.cpp [hunk 1]
```diff
@@ -553,12 +553,12 @@ void YGNode::cloneChildrenIfNeeded() {
   }
 }
 
-void YGNode::markDirtyAndPropogate() {
+void YGNode::markDirtyAndPropagate() {
```
Comment (janicduplessis) [inline ReactCommon/yoga/yoga/YGNode.cpp]: The changes in the yoga folder need to be made to https://github.com/facebook/yoga then they are synced here.

Comment (ss18) [reply ReactCommon/yoga/yoga/YGNode.cpp]: @jacobrosenthal in my prev PR it was done automatically https://github.com/facebook/react-native/pull/17578

Comment (ss18) [reply ReactCommon/yoga/yoga/YGNode.cpp]: also Yoga has been changed, why not do the same in this PR?

Comment (ide) [reply ReactCommon/yoga/yoga/YGNode.cpp]: If the changes were applied to the yoga repo they’ll eventually make their way to RN sometime.

Comment (ss18) [reply ReactCommon/yoga/yoga/YGNode.cpp]: @ide 🤦‍♂️

**Comment by shergin** [comment]
@ss18 Could you please resolve conflicts?

**Comment by ss18** [comment]
@shergin done.

**Comment by shergin** [comment]
@ss18 I am sorry, could you please remove all Yoga changes from the PR. Apparently, it's public API and it's impossible to land without massive codemod across internal FB codebase. I will address it separately later.

**Comment by ss18** [comment]
@shergin done!

**Pull request closed without merging**